dockstore-webservice/src/main/java/io/dockstore/webservice/core/WorkflowVersion.java

```diff
@@ -268,20 +269,34 @@ public void setKernelImagePath(String kernelImagePath) {
     @ApiModel(value = "WorkflowVersionPathInfo", description = "Object that "
             + "contains the Dockstore path to the workflow and the version tag name.")
     public static final class WorkflowVersionPathInfo {
-        @ApiModelProperty(value = "Dockstore path to workflow.")
+        @ApiModelProperty(value = "Dockstore path to workflow")
```

Comment:
FYI, these changes and the new `ApiModelProperty` won't effectively do anything, since they're Swagger annotations and we don't generate swagger.yaml any more. TLDR; change to `@Schema` (I think that's it)